src/table/virtualized-table/hooks/use-dynamic-row-height.ts

```diff
@@ -77,14 +77,14 @@ const useDynamicRowHeight = ({
   );
 
   const setCellSize = useCallback(
-    (text: string, rowIdx: number, colIdx: number, batchStateUpdate = false) => {
+    (text: string, rowIdx: number, colIdx: number, isNumeric = false, batchStateUpdate = false) => {
```

Comment:
if you have some tests cases of this function specifically for `batchStateUpdate`, then those might be affected by addition of `isNumeric` argument here

Reply:
Good catch! There was not any test affect by it, but it was something that I forgot to check